Formula/libtommath.rb

```diff
@@ -28,10 +30,10 @@ class Libtommath < Formula
   def install
     ENV["DESTDIR"] = prefix
 
-    system "make"
-    system "make", "test_standalone"
+    system "make", "-f", "makefile.shared"
+    system "make", "test"
     include.install Dir["tommath*.h"]
-    lib.install "libtommath.a"
+    lib.install Dir[".libs/libtommath*.a", ".libs/#{shared_library("libtommath", "*")}"]
```

Comment:
Actually, now that I've had a look at `makefile` and `makefile.shared`, is there any reason we don't do `make install` here?